### PR TITLE
add a limit on the length of the resource name

### PR DESCRIPTION
**Pull request by tpiperatgod**
for PR https://github.com/streamnative/function-mesh/pull/437

Signed-off-by: laminar <tpiperatgod@gmail.com>

### CODE REVIEW

**Review by freeznet — APPROVED** [review]

File: docs/connectors/io-crd-config/sink-crd-config.md [hunk 1]
```diff
@@ -11,7 +11,7 @@ This table lists sink configurations.
 
 | Field | Description |
 | --- | --- |
-| `name` | The name of a sink connector. |
+| `name` | The name of a sink connector, length is limited to `43` characters. |
```
Comment (Huanli-Meng) [inline docs/connectors/io-crd-config/sink-crd-config.md:14]: The connector name is a string of up to 43 characters.

**Review by Huanli-Meng — COMMENTED** [review]
leave a minor comment. 
Do we need to update docs in other releases? If yes, please help update the docs accordingly. Thanks.

**Review by nlu90 — DISMISSED** [review]

**Comment by Huanli-Meng** [comment]
@tpiperatgod Could you please help check my comment? Thanks.

**Comment by tpiperatgod** [comment]
> @tpiperatgod Could you please help check my comment? Thanks.

done...

**Review by Huanli-Meng — APPROVED** [review]

**Review by freeznet — APPROVED** [review]

**Review by tuteng — APPROVED** [review]

**Pull request merged**